libipl/p10/ipl0.C

```diff
@@ -157,6 +159,15 @@ static int update_hwas_state_callback(struct pdbg_target* target, void *priv)
 
 	} else if (ipl_type() == IPL_TYPE_NORMAL) {
 
+		if (type == TGT_TYPE_PROC) {
+			if (ipl_is_master_proc(target)) {
+				ipl_log(IPL_INFO,
+					"Primary processor is guarded "
+					"so, skipping to apply");
+				return GUARD_PRIMARY_PROC_NOT_APPLIED;
```

Comment:
where are you checking this return type ?

Reply:
We are not checking this return code anywhere.

In LN 266, there is already a check for error cases (return codes 0 and 2) but, this is not the error case